src/gwt/src/org/rstudio/studio/client/workbench/views/console/Console.java

```diff
@@ -174,6 +180,23 @@ else if (StringUtil.equals(type, ReticulateEvent.TYPE_REPL_BUSY))
          else
             view.leaveMode(ConsoleMode.Job);
       });
+
+      events.addHandler(SessionSerializationEvent.TYPE, event -> {
+         if (event.getAction().getType() == SessionSerializationAction.SUSPEND_SESSION)
+         {
+            view.getSuspendBlockedIcon().asWidget().setVisible(false);
+            view.getSuspendedIcon().asWidget().setVisible(true);
+         }
+         else if (event.getAction().getType() == SessionSerializationAction.RESUME_SESSION)
+         {
+            view.getSuspendBlockedIcon().asWidget().setVisible(false);
+            view.getSuspendedIcon().asWidget().setVisible(false);
+         }
+      });
+
+      events.addHandler(SessionSuspendBlockedEvent.TYPE, event -> {
+         view.setSuspendBlockedIcon(event);
+      });
```

Comment:
Oops, I forgot to point this one out:
```suggestion
      events.addHandler(SessionSerializationEvent.TYPE, event -> {
         view.onSerializationEvent(event);
      });

      events.addHandler(SessionSuspendBlockedEvent.TYPE, event -> {
         view.onSuspendedBlockedEvent(event);
      });
```